config/default.yml

```diff
@@ -4,6 +4,10 @@ AllCops:
     Patterns:
     - _spec.rb
     - "(?:^|/)spec/"
+  FactoryGirl:
```

Comment:
Thanks, @Darhazer.

Should I also move the Cop class to `lib/rubocop/cop/rspec/factory_girl` and make the module with 4 levels like `RuboCop::Cop::RSpec::FactoryGirl`?